Services/CO.CDP.Organisation.WebApi/Model/Command.cs

```diff
@@ -181,6 +181,25 @@ public record RegisterConnectedEntity
     public DateTimeOffset? EndDate { get; init; }
 }
 
+public record UpdateConnectedEntity
+{
+    public string Id { get; init; }
```

Comment:
Static analysis reports a problem here: `Non-nullable property 'Id' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.`.